### DISCUSSION TITLE
AD LDAP Issue please help

### DISCUSSION THREAD

**Discussion by EnshengChu** [goharbor/harbor#15723] [Q&A]
I am using AD and I want to use LDAP, but wen I setting up the LDAP  and, after I login thins fist account of AD than the rest account can not login , log always show this massage all the time...

Oct  1 16:30:30 172.19.0.1 core[1171]: 2021-10-01T16:30:30Z [ERROR] [/core/auth/authenticator.go:262]: Failed to get user from DB, username: test1, error: user test1 not found
Oct  1 16:30:30 172.19.0.1 core[1171]: 2021-10-01T16:30:30Z [ERROR] [/core/controllers/base.go:103]: Error occurred in UserLogin: user test1 or email  already exists

this is my config setting

LDAP URL: ldap://172.10.0.40:389
LDAP Search DN: CN=Angus Chu,OU=Users,OU=Anguslab,DC=anguslab,DC=tk
LDAP Base DN : dc=anguslab,dc=tk
LDAP UID : sAMAccountName
LDAP Scope : subtree

 Is anyone had experience on this issue ?
help me please

**Comment by vemundd** [comment]
+1
I'm experiencing the exact same issue with LDAPS. I have no solution, though

**Reply by stonezdj** [reply]
Check if is there any duplicated email address with the user test1.
```
docker exec -it harbor-db bash
psql -U postgres -d registry
select * from harbor_user
```
